install/scripts/start-ec.sh

```diff
@@ -123,7 +139,27 @@ if [ "$CLIENT" = "nethermind" ]; then
     # Uncomment peer report logging restrictions in the log config XML
     sed -i 's/<!-- \(<logger name=\"Synchronization\.Peers\.SyncPeersReport\".*\/>\).*-->/\1/g' /nethermind/NLog.config
 
+<<<<<<< HEAD
+
+    CMD="$PERF_PREFIX /nethermind/Nethermind.Runner \
+      --config $NETHERMIND_NETWORK \
+      --datadir /ethclient/nethermind \
+      --JsonRpc.Enabled true \
+      --JsonRpc.Host 0.0.0.0 \
+      --JsonRpc.Port ${EC_HTTP_PORT:-8545} \
+      --JsonRpc.EnginePort ${EC_ENGINE_PORT:-8551} \
+      --JsonRpc.EngineHost 0.0.0.0 \
+      --Sync.AncientBodiesBarrier 1 \
+      --Sync.AncientReceiptsBarrier 1 \
+      --Sync.SnapSync true \
+      --Merge.Enabled true \
+      --JsonRpc.JwtSecretFile=/secrets/jwtsecret \
+      $EC_ADDITIONAL_FLAGS"
+||||||| 99a3e21
```

Comment:
Looks like you were in the middle of a merge conflict when you pushed this, can you resolve it?

Reply:
Missed this one. fixed